src/Reflection/ReflectionNamedType.php

```diff
@@ -54,6 +64,47 @@ public function isBuiltin(): bool
         return array_key_exists(strtolower($this->name), self::BUILT_IN_TYPES);
     }
 
+    public function getClass(): ReflectionClass
+    {
+        if (
+            ! $this->owner instanceof ReflectionEnum
```

Comment:
Can some of these conditionals be inverted to lead to early returns? For example, the `$this->isBuiltIn()` can be moved up, and `return $this->reflector->reflectClass($this->name);` too, assuming we can make these a happy path branch?

Reply:
Thanks. I was not happy with the previous version. Now it's much more beutiful.